torchsynth/synth.py

```diff
@@ -327,6 +364,9 @@ def __init__(self, synthconfig: Optional[SynthConfig] = None, *args, **kwargs):
             ]
         )
 
+        # Load the nebula
+        self.load_hyperparameters(nebula)
```

Comment:
I kind of want this to be in AbstractSynth instead of in Voice. But, we don't want to force loading of hyperparams -- and I'm also not sure where it should be called since it needs to occur after all the modules and params are added.

Reply:
Yeah nebulae are specific to specific synths. But I agree it could go into AbstractSynth somehow. For later: https://github.com/torchsynth/torchsynth/issues/321